src/cli.rs

```diff
@@ -1024,6 +1024,14 @@ pub fn cli() -> Command {
                 .required(false)
                 .long("dot")
                 .help("Output the dependency DAG in the Graphviz DOT format")
+                .conflicts_with("serial-buildorder")
+            )
+            .arg(Arg::new("serial-buildorder")
+                .action(ArgAction::SetTrue)
+                .required(false)
+                .long("serial-buildorder")
+                .help("Output the tree-of package in a serial build order")
```

Comment:
`tree-of package` seems wrong in that sentence...

We probably want something like "Print the dependencies in a serial build order (flattened tree)"

Ideally even with a long description that explains what "serial build order" means (basically a reversed topological ordering where we list/build the dependencies/children first).

Maybe "sequential" is even a better term than "serial"?

Reply:
I find the term "serial" more fitting for this (and the flag is shorter), but I believe "serial" and "sequential" could be used interchangeably here.

Will add a long help version along the lines of
> A serial build order ensures that dependencies are built one at a time in a specific sequence.